src/components/sandbox/Sandbox.tsx

```diff
@@ -195,12 +195,11 @@ function ComponentList(props: ComponentListProps): ReactElement[] {
   const children = React.Children.toArray(props.sandboxProps.children) as ReactElement[];
 
   return children.filter(el =>
-    React.isValidElement(el)
-    && typeof el.type === 'function'
-    && (el.type.name.toLowerCase().startsWith(props.type)
-        || props.sandboxProps.allow?.includes(el.type.name)
+    React.isValidElement(el) && (
+      (typeof el.type === "function" && el.type.name.toLowerCase().startsWith(props.type)) ||
+      (typeof el.type === "string" && props.sandboxProps.allow?.includes(el.type))
     )
-  );
+  )
```

Comment:
What is the reason for the change in logic here?

Reply:
It's fine as is. It was just when I was first looking at the code in split view it was hard to read and looked a lot more complex than it was.